<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1687</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kasa Desktop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="bills_table">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>36</y>
      <width>1021</width>
      <height>711</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Category</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Initial Value</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Payment Day</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1687</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionNew_User"/>
   </widget>
   <widget class="QMenu" name="menuBills">
    <property name="title">
     <string>Bills</string>
    </property>
    <addaction name="actionLoad_Bills"/>
    <addaction name="actionNew_Bill"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuBills"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionNew_User">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>List</string>
   </property>
  </action>
  <action name="actionLoad_Bills">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionNew_Bill">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
